ts/test/test-profiler.ts

```diff
@@ -637,7 +637,9 @@ describe('Profiler', () => {
                            .onCall(0)
                            .callsArgWith(1, undefined, undefined, {
                              statusCode: 409,
-                             body: {error: {details: [{retryDelay: '50s'}]}}
+                             body: {
+                               message: 'action throttled, backoff for 50s',
```

Comment:
Add a test for a string like "1m2s"? The backoff duration in the message is in the form "72h3m0.5s".

Reply:
Here are test cases I used elsewhere:

```java
    assertThat(tryParseBackoffDuration("")).isEqualTo(0.0);
    assertThat(tryParseBackoffDuration("action throttled, backoff for 1m2s")).isEqualTo(62.0);
    assertThat(tryParseBackoffDuration("action throttled, backoff for 2.5s")).isEqualTo(2.5);
    assertThat(tryParseBackoffDuration("action throttled, backoff for 1m2.5s")).isEqualTo(62.5);
    assertThat(tryParseBackoffDuration("action throttled, backoff for 10m0s")).isEqualTo(600.0);
    assertThat(tryParseBackoffDuration("action throttled, backoff for 1h2.5s")).isEqualTo(3602.5);
    assertThat(tryParseBackoffDuration("action throttled, backoff for 1h1m2.5s")).isEqualTo(3662.5);
    assertThat(tryParseBackoffDuration("action throttled, backoff for 1m2+s")).isEqualTo(0.0);
```

Reply:
Added test cases matching these.